[2022-08-19 23:10:03,517] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T05:23:00+00:00 [queued]>
[2022-08-19 23:10:03,558] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T05:23:00+00:00 [queued]>
[2022-08-19 23:10:03,564] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 23:10:03,568] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-19 23:10:03,572] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 23:10:03,645] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): get_telegram_version> on 2020-01-01 05:23:00+00:00
[2022-08-19 23:10:03,670] {standard_task_runner.py:52} INFO - Started process 3683 to run task
[2022-08-19 23:10:03,696] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'sefa_dag', 'get_telegram_version', 'scheduled__2020-01-01T05:23:00+00:00', '--job-id', '673', '--raw', '--subdir', 'DAGS_FOLDER/sefa_dag.py', '--cfg-path', '/tmp/tmprjgokk44', '--error-file', '/tmp/tmphgldpqec']
[2022-08-19 23:10:03,700] {standard_task_runner.py:80} INFO - Job 673: Subtask get_telegram_version
[2022-08-19 23:10:03,848] {task_command.py:371} INFO - Running <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T05:23:00+00:00 [running]> on host fd7a784b9251
[2022-08-19 23:10:04,070] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=sefa_dag
AIRFLOW_CTX_TASK_ID=get_telegram_version
AIRFLOW_CTX_EXECUTION_DATE=2020-01-01T05:23:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-01T05:23:00+00:00
[2022-08-19 23:10:04,078] {python.py:173} INFO - Done. Returned value was: 1.24.0
[2022-08-19 23:10:04,123] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=sefa_dag, task_id=get_telegram_version, execution_date=20200101T052300, start_date=20220819T231003, end_date=20220819T231004
[2022-08-19 23:10:04,222] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-19 23:10:04,285] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
